assets/js/script.js

```diff
@@ -295,7 +309,7 @@ submitBTN.addEventListener('click', GetBpmApi)
 
 function createArrObj(inputData){
     var arrayOfObjects = [];
-    for(i=0; i < inputData.length; i++) {
+    for(i=0; i < inputData.tempo.length; i++) {
```

Comment:
This is where the minor bug was. We needed to access the tempo property first.

Reply:
sounds good. Everything else works so far